frameworks/kafka/docs/security.md

```diff
@@ -170,14 +124,14 @@ Install the DC/OS Apache Kafka service with the following options in addition to
     }
 }
 ```
-The DC/OS Apache ZooKeeper service is intended for this purpose and supports Kerberos.
+The DC/OS Apache ZooKeeper service, package `kafka-zookeeper`, is intended for this purpose and supports Kerberos.
```

Comment:
What about: "The DC/OS Apache ZooKeeper service (`kafka-zookeeper` package)" instead?

Since this will have to be updated for Confluent, we may want to template these too?

Reply:
I think I will probably just use a mangle for now...